### PR TITLE
App Styling Milestone: Styling using TailwindCSS

### PR DESCRIPTION
**Pull request by AyoubM490**
In this milestone, I:
- Used TailwindCSS
- Copied the styles from zeplin

### CODE REVIEW

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Ayoub, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] there are no files changed in this pull request so I'm not able to leave inline comments, however, your application doesn't seem to work due to the below piece of code
https://github.com/AyoubMs/Bookstore-react/blob/ef0a8bfd7d38d36ca323bb6008e09bbf7c22472d/src/redux/configureStore.js#L16-L25
when removing lines 21 & 22 the application runs as it is supposed to, you are also not supposed to turn off linters these are used to make your application run better and look cleaner. please fix this issue.
- [x] the link to your MIT license is broken, please make sure it is referring to the right file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -13,16 +13,12 @@ const rootReducer = combineReducers({
...
 });
 
-/* eslint-disable no-underscore-dangle */
 const store = createStore(
   rootReducer,
   compose(
     applyMiddleware(thunk),
-    window.__REDUX_DEVTOOLS_EXTENSION__
-      && window.__REDUX_DEVTOOLS_EXTENSION__(),
   ),
```
Comment (micheaol) [inline src/redux/configureStore.js:20]: Please ensure that your final design should match Zeplin's design. 

Your Implementation | Zeplin's design
:---:|:---:
image | image

Please use the information detailed in Zeplin to style the [website](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) to get the exact match of the design.

image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @AyoubMs , 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by medaminedev66 — APPROVED** [review]
## Hi @AyoubMs, 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**